nc_test4/Makefile.am

```diff
@@ -111,7 +111,7 @@ foo1.nc tst_*.h4 test.nc testszip.nc test.h5 szip_dump.cdl		\
 perftest.txt bigmeta.nc bigvars.nc *.gz MSGCPP_*.nc	                \
 floats*.nc floats*.cdl shorts*.nc shorts*.cdl ints*.nc ints*.cdl        \
 testfilter_reg.nc filterrepeat.txt tmp_fillonly.nc                      \
-testfilter_order.nc crfilterorder.txt rdfilterorder.txt
+testfilter_order.nc crfilterorder.txt rdfilterorder.txt 1
```

Comment:
Is this "1" intentional? Is there a file named "1"?

Reply:
Yes, and yes. At a glance I couldn’t determine where it was coming from, so it’s on the agenda as a detail to follow up with at the next team meeting.

Reply:
It should be a typo. Not sure how a file named "1" got in.

Reply:
Nor I, and weirdly enough, grepping for '1' did not turn up any *useful* information XD.  We'll track it down before the forthcoming release.

Reply:
I got it. It is occurring in nc_test4/tst_misc.sh. This line, I think:
> if $NCDUMP ./tst_charvlenbug.nc 2>1 >./tmp ; then

It was an attempt to send stderr to stdout. But I got it wrong. I think it should be:
> if $NCDUMP ./tst_charvlenbug.nc >& ./tmp ; then